openproblems/tasks/spatial_decomposition/methods/baseline.py

```diff
@@ -0,0 +1,40 @@
+from ....tools.decorators import method
+from ....tools.utils import check_version
+from ..utils import split_sc_and_sp
+
+import numpy as np
+
+
+@method(
+    method_name="Random Proportions",
+    paper_name="Random Proportions (baseline)",
+    paper_url="https://openproblems.bio",
+    paper_year=2022,
+    code_url="https://github.com/openproblems-bio/openproblems",
+    is_baseline=True,
+)
+def random_proportions(adata, test=False):
+    adata_sc, adata = split_sc_and_sp(adata)
+    label_distribution = adata_sc.obs["label"].value_counts()
+    adata.obsm["proportions_pred"] = np.random.dirichlet(
+        label_distribution,
+        size=adata.shape[0],
```

Comment:
is the adata.shape[0] the number of cells or cell types here?

Reply:
And you predict the proportion of cell types per cell? I don't understand the output format I guess.

Reply:
*n_voxels, n_cell_types, sorry. For each sample of spatial data we predict proportion of each cell type. adata.shape[0] is n_voxels.

Reply:
okay, makes more sense :). So `adata` is the spatial `adata` then I guess. I'm good with the PR as is :)